### ISSUE TITLE
Button labels not unique on page when have same label. Lack of label means to name unique with off-screen text (2.4.6)

### ISSUE THREAD

**Issue by Blackbaud-MattGregg** [blackbaud/skyux-theme#135]
### Issue from RE NXT accessibility audit 2019

**Ensure form field labels are unique**

Form controls including button menu elements such as "Edit" are not unique.

[Code]
```
<button class="btn bb-btn-secondary bb-constituentui-summary-details-update-contact-information-button ng-binding dropdown-toggle" uib-dropdown-toggle="" type="button" aria-haspopup="true" aria-expanded="false">
Edit <span class="caret"></span>
</button>
```

### Impact
In the absence of contextual information, users of assistive technology may not identify or understand the purpose of the buttons.

### Recommendation
Developers must add contextual information to the buttons using off-screen text.

[Compliant Code]
```
<button class="btn bb-btn-secondary bb-constituentui-summary-details-update-contact-information-button ng-binding dropdown-toggle" uib-dropdown-toggle="" type="button" aria-haspopup="true" aria-expanded="false">
Edit <span class="off-screen"> Constituent Summary</span>
</button>
```

If there is a way for teams to add off-screen contextual information for their button labels. E.g., when there are multiple Edit or other duplicate buttons on a page, they can differentiate them with additional text and have this be off-screen, this then just needs to be a documentation issue to add to Button and Dropdown docs page.

**Comment by Blackbaud-DanMiller** [comment]
#44

**Comment by Blackbaud-SteveBrush** [comment]
You can now use the `.sky-screen-reader-only` CSS class.
```
<button class="sky-btn sky-btn-default">
Edit <span class="sky-screen-reader-only">Constituent Summary</span>
</button>
```
